dfget/core/api/supernode_api.go

```diff
@@ -100,9 +105,14 @@ func (api *supernodeAPI) ReportPiece(node string, req *types.ReportPieceRequest)
 
 	url := fmt.Sprintf("%s://%s%s?%s",
 		api.Scheme, node, peerReportPiecePath, util.ParseQuery(req))
-
 	resp = new(types.BaseResponse)
 	e = api.get(url, resp)
+	if e != nil {
+		logrus.Infof("failed to report piece{taskid:%s,range:%s},err: %v", req.TaskID, req.PieceRange, e)
```

Comment:
Please confirm that here is an `Infof` level log, or `Errorf` level log?

Reply:
https://github.com/dragonflyoss/Dragonfly/pull/682#discussion_r305192168

Reply:
@Starnop https://github.com/dragonflyoss/Dragonfly/pull/682#discussion_r305192168
Error or Info ?

Reply:
I am sure that `Info` level is improper. Both `Error` and `Warn` are better.

Reply:
> I am sure that `Info` level is improper. Both `Error` and `Warn` are better.
I am agree with you.I will modify it

Reply:
Done